rules/backend.txt

```diff
@@ -261,3 +261,13 @@ eval(str_rot13
 # max chadwick for president
 # wordpress backdoor
 $db___WP=create_function
+
+# create file from request
+$r=$_REQUEST;@file_put_contents($r[
+
+# direct eval of request
+eval($_REQUEST[
```

Comment:
Ouch indeed, did not think of that. I would indeed have made it a regex if I did.
I knew it was a bit short but I thought "well, direct evil of $_REQUEST is always a bad thing".